src/main/java/com/minelittlepony/bigpony/mod/gui/GuiBigSettings.java

```diff
@@ -17,102 +16,86 @@
-        this.buttonList.add(fillyPreset = new GuiButton(15, 220, 120, 120, 20, "Filly"));
+
+        cameraPresets = new CameraPresetButton[values.length];
+        scalePresets = new CameraPresetButton[values.length];
+        combinedPresets = new CameraPresetButton[values.length];
+
+        for (int i = 0; i < cameraPresets.length; i++) {
+            buttonList.add(cameraPresets[i] = new CameraPresetButton(this, values[i], right + 100, 40, true, false));
+            buttonList.add(scalePresets[i] = new CameraPresetButton(this, values[i], right + 80, 40, false, true));
+            buttonList.add(combinedPresets[i] = new CameraPresetButton(this, values[i], right, 40, true, true));
+        }
     }
 
     @Override
     public void drawScreen(int mouseX, int mouseY, float partialTicks) {
         this.drawDefaultBackground();
         super.drawScreen(mouseX, mouseY, partialTicks);
-        this.drawCenteredString(this.fontRenderer, "BigPony settings", width / 2, 10, -1);
-        this.drawCenteredString(this.fontRenderer, "Camera Presets", 280, 25, -1);
+        
+        int left = width / 2 - 200;
+        int right = width / 2 + 16;
+        
+        drawCenteredString(fontRenderer, I18n.format("minebp.options.title"), width / 2, 10, -1);
+        drawCenteredString(fontRenderer, I18n.format("minebp.options.body"), left + 80, 25, -1);
+        drawCenteredString(fontRenderer, I18n.format("minebp.options.camera"), left + 80, 125, -1);
+        drawCenteredString(fontRenderer, I18n.format("minebp.options.presets"), right + 90, 25, -1);
     }
 
     @Override
     public void updateScreen() {
-
-        resetXSize.enabled = !FloatUtils.equals(xSize.getSliderValue(), 1);
-        resetYSize.enabled = !FloatUtils.equals(ySize.getSliderValue(), 1);
-        resetZSize.enabled = !FloatUtils.equals(zSize.getSliderValue(), 1);
-        resetHeight.enabled = !FloatUtils.equals(height.getSliderValue(), 1);
-        resetDistance.enabled = !FloatUtils.equals(distance.getSliderValue(), 1);
-
-        humanPreset.enabled = !CameraPresets.HUMAN.isEqual(height.getSliderValue(), distance.getSliderValue());
-        marePreset.enabled = !CameraPresets.MARE.isEqual(height.getSliderValue(), distance.getSliderValue());
-        stallionPreset.enabled = !CameraPresets.STALLION.isEqual(height.getSliderValue(), distance.getSliderValue());
-        princessPreset.enabled = !CameraPresets.ALICORN.isEqual(height.getSliderValue(), distance.getSliderValue());
-        fillyPreset.enabled = !CameraPresets.FILLY.isEqual(height.getSliderValue(), distance.getSliderValue());
+        CameraPresets[] values = CameraPresets.values();
+        for (int i = 0; i < cameraPresets.length; i++) {
+            cameraPresets[i].enabled = !values[i].isEqual(height.getSliderValue(), distance.getSliderValue());
+            scalePresets[i].enabled = !values[i].isEqual(xSize.getSliderValue(), ySize.getSliderValue(), zSize.getSliderValue());
```

Comment:
Does this work?

Reply:
It wouldn't be in the PR if it didn't.

Reply:
@killjoy1221 You pose a very convincing argument. Feature added. :3